backend/src/src-moderation/src/main/java/com/siemens/sw360/moderation/db/ModerationDatabaseHandler.java

```diff
@@ -280,24 +298,36 @@ public void createRequest(User user) {
         return moderators;
     }
 
-    private Set<String> getAdministrators(String department) {
+    private Set<String> getUsersAtLeast(UserGroup userGroup, String department) {
         List<User> sw360users = getAllSW360Users();
-        List<User> allAdministrators = sw360users
+        List<User> allRelevantUsers = sw360users
                     .stream()
-                    .filter(user1 -> PermissionUtils.isUserAtLeast(UserGroup.ADMIN, user1))
+                    .filter(user1 -> PermissionUtils.isUserAtLeast(userGroup, user1))
                     .collect(Collectors.toList());
-        List<User> administrators = allAdministrators.stream()
+        List<User> relevantUsersOfDepartment = allRelevantUsers.stream()
                     .filter(user -> user.getDepartment().equals(department))
                     .collect(Collectors.toList());
-        if (administrators.isEmpty()){
-            // no admins for department found -> fall back to all admins
-            administrators = allAdministrators;
-        }
-        Set<String> adminEmails = administrators.stream()
+
+        List<User> resultingUsers = relevantUsersOfDepartment.isEmpty() ? allRelevantUsers : relevantUsersOfDepartment;
+        Set<String> resultingUserEmails = resultingUsers.stream()
                     .map(User::getEmail)
                     .collect(Collectors.toSet());
 
-        return adminEmails;
+        return resultingUserEmails;
+    }
+
+    private Set<String> getUsersAtLeast(UserGroup userGroup) {
```

Comment:
Ok, I let one method call the other. But I have to treat the null case separately, since at least the setup-admin has department _null_.